controllers/ContactUsController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Request\ContactUsRequest;
+// use Illuminate\Http\Request;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Contracts\View\View;
+use Illuminate\Mail\Message;
+
+class ContactUsController extends Controller
+{
+    public function view(){
+        return view('contacts.contactUs');
+    }
+
+    public function send(ContactUsRequest $request): RedirectResponse
+    {
+        // dd($request->validated());
+
+        $callback = function (string $test): string {
```

Comment:
please be attentive on Monday lesson it was removed in the end.

Reply:
commented $callback  https://github.com/liubamiron/bootcamp-project/pull/9/commits/f91173f1c42186badb07ca9da1c9b0879198c481

Reply:
Should be removed and not commented. Will ignore as it is removed in the next pr.